ui/build.gradle

```diff
@@ -0,0 +1,100 @@
+apply from: '../versions.gradle'
+apply plugin: 'com.android.library'
+
+ext {
+    PUBLISH_GROUP_ID = 'com.xmartlabs'
+    PUBLISH_ARTIFACT_ID = 'ui'
+    PUBLISH_VERSION = '0.1'
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'com.jakewharton.butterknife'
+
+buildscript {
+    repositories {
+        jcenter()
+        maven { url 'https://maven.fabric.io/public' }
+    }
+    dependencies {
+        classpath "com.jakewharton:butterknife-gradle-plugin:8.2.1"
+        classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
```

Comment:
I agree. Somehow it can't be done the way it was with the latter changes @diegomedina248 did. We need to see in what other way we can do that.